scripted-sbt-redux/src/main/scala/sbt/scriptedtest/ScriptedTests.scala

```diff
@@ -492,9 +492,13 @@ class ScriptedRunner {
       get(tests, baseDir, accept, logger).map(st => (st.group, st.name))
     val scriptedRunners =
       runner.batchScriptedRunner(scriptedTests, addTestFile, instances, prop, logger)
-    val parallelRunners = scriptedRunners.toParArray
-    parallelRunners.tasksupport = new ForkJoinTaskSupport(new ForkJoinPool(instances))
-    runAll(parallelRunners)
+    if (instances > 1 && instances != Int.MaxValue) {
+      val parallelRunners = scriptedRunners.toParArray
+      parallelRunners.tasksupport = new ForkJoinTaskSupport(new ForkJoinPool(instances))
+      runAll(parallelRunners)
+    } else {
+      runAll(scriptedRunners)
+    }
```

Comment:
Minor nit pick. `-1` may be a better value for "I'm gonna do sequential even tho the method is called runInParallel" since I'd expect `Int.MaxValue` to go max. Scaladoc explaining this would be nice. Overall LGTM.

Reply:
Yeah, Int.MaxValue wasn't a perfect sentinel. I reworked this to instead make a new `run` variant that has a `parellelExecution` flag rather than using a sentinel.